feature/header

IrenMarkova opened the pull request:
Добавила Хедер и Модалку

IrenMarkova committed:
Добавила Хедер и Модалку

yulikafsd commented on src/partials/header.html:
```diff
@@ -0,0 +1,37 @@
+        <li class="header__item"><a href="#contacts" class="link header__link">Contact</a></li>
+      </ul>
+    </nav>
+    <button type="button" class="burger burger__header js-open-menu" aria-label="burger menu">
```
`aria-label="burger menu"` - лучше в ариа лейбл что-то более понятное юзеру написать типа open menu button

IrenMarkova replied:
ok

yulikafsd commented on src/sass/layout/_header.scss:
```diff
@@ -0,0 +1,93 @@
+.header__container {
+  // outline: 1px solid orange;
+  //   padding-bottom: 21px;
+  padding-top: 6px;
```
верхние и нижние паддинги задаются секции, не контейнеру. контейнеру задаются только боковые паддинги и только если они отличаются от заданных классу контейнер по умолчанию в базовых стилях

IrenMarkova replied:
оk - добавила доп div header-box  и падинги верхние перенесла в него, так меньше кода - чем добавлять каждой части внутри

yulikafsd commented on src/sass/layout/_header.scss:
```diff
@@ -0,0 +1,93 @@
+.header__list {
+  display: none;
+  @include minMedia($desk) {
+    margin-left: 110px;
```
насчет маржинов, задавай, пожалуйста, элементам или всем левые, или всем правые, иначе легко запутаться. то же самое с паддингами - или всем нижние, или всем верхние

IrenMarkova replied:
ok, margin-left установила, падинги былы только верхние, нижний использовала для теста - он закомичен.

IrenMarkova committed:
Изменила логотип и внесла правки

1) В логотипе добавлена анимация
2) Правки согласно замечаний
3) Иконку меню при ховере фокусе заменила на эффект увеличения, изменение цвета в палитре макета нет подходящего
4) в модалке overflow изменила на auto

IrenMarkova commented:
1) В логотипе добавлена анимация
2) Правки согласно замечаний
3) Иконку меню при ховере фокусе заменила на эффект увеличения, изменение цвета в палитре макета нет подходящего
4) в модалке overflow изменила на auto

IrenMarkova committed:
добавила закрытие symbol в спрайте

IrenMarkova committed:
Update _modal.scss

убрала в модалке высоту вюпорта

IrenMarkova committed:
исправила размер иконки в кнопке

IrenMarkova committed:
добавила в модалке цвет курсора в input

IrenMarkova committed:
добавила цвет курсора в модалке в блоке комментарии

IrenMarkova committed:
корректировка падингов секции header

yulikafsd committed:
Merge branch 'main' into feature/header
